code/mob/living/carbon/human/procs/get_desc.dm

```diff
@@ -410,6 +410,30 @@
 		items = copytext(items, 3)
 		. += "<br>[SPAN_NOTICE("[src] is juggling [items]!")]"
 
+	if (src.reagents.has_reagent("ethanol"))
+		if (usr.mind && usr.mind.assigned_role == "Bartender")
```

Comment:
I don't like things checking for your job.
These should check if you have bartender training.

training_drinker is also shared with HoS and det. Giving at least the last ability to these would make sense (honestly that one could be universal). Training specifically for bartender could be made if we do not want to give the other abilities to HoS and det.

Reply:
I also vote for just slapping it on training_drinker and giving that to the Bartender. It fits.

Reply:
I'm not sure about putting the first two abilities on training_drinker-- the HoS and Detective won't benefit nearly as much from the information as the bartender will, and I feel it'd just clog up the reagent scans with useless information. I'd rather just make Bartender Training. The third ability being for everyone makes sense, I'll implement that.